bag/run-etl.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Script for running the complete ETL process using data from the reference database.
+# The data resulting from this process is served through the REST API
+# included in this repository.  
+#
+# The process entails:
+#   - loading data from the reference database into a separate schema (e.g. bag_v11)
+#   - using the loaded data to create/update the indices in Elasticsearch
+#
+# TODO: Note that currently this will incur downtime since we are truncating and writing
+# directly to the target databases.
+
```

Comment:
```suggestion
set -e -x
```

(?)

Reply:
i agree with -e but -x usually just makes logs more unreadable imho

Reply:
Agree, you already have a bunch of echos.